connectors/trpc/README.md

```diff
@@ -83,16 +84,20 @@ const result = await client.run({
 });
 ```
 
-## Notes
+## Technical Details
+
+The plugin does two things:
+
+1. It adds a `meta` field to the procedures with `{ inferable: { enabled: true } }`. This is used to identify the procedures that should be exposed as Inferable functions.
+2. It adds a `ctx` field to the tRPC procedure so you can validate the context that's passed by Inferable in your down stream procedures or middleware.
 
-- Preserve Middleware: All your existing tRPC middleware continues to work
-- Type Safety: Maintains full type safety through your tRPC router
-- Selective Exposure: Only procedures marked with meta({ inferable: true }) are exposed
-- Custom Descriptions: Add descriptions to help guide the AI through meta({ description: "..." })
+- This allows you model [human in the loop](https://docs.inferable.ai/pages/human-in-the-loop) workflows where you can fire off a approval request to a human before the function is run.
```

Comment:
I don't think we have a mechanism to support this in tRPC?

You would need to return an `interrupt` result type?

Reply:
Yeah. It works the same way.
![CleanShot 2024-12-11 at 10 12 19@2x](https://github.com/user-attachments/assets/a6c3c2eb-ae0d-42d5-ac99-2fc445757fb8)